Gemfile

```diff
@@ -18,4 +18,6 @@ gem 'mysql2'
 gem 'sqlite3'
 gem 'pg'
 
+gem 'shipwire', github: 'nebulab/shipwire'
```

Comment:
This won't be included in the final version of the gem. I think we need to figure out if we need to release a separate gem while they merge code in.